R/plot_land.R

```diff
@@ -75,7 +75,13 @@ plotRegionalLandAllocation <- function(aScenarioInfo) {
     summarize(land.allocation = sum(land.allocation)) ->
     regionalLand
 
+  # Define a color palette (default for ggplot2 is rainbow which is hard to read)
+  my_palette <- c('#a6cee3','#1f78b4','#b2df8a','#33a02c','#fb9a99','#e31a1c','#fdbf6f','#ff7f00','#cab2d6','#6a3d9a','#ffff99','#b15928',
```

Comment:
Generally, I recommend `scale_fill_brewer` for this sort of thing.  The `ggthemes` package also has some color scales that are meant to go with certain themes.  There is a solarized theme/scale, and there are themes/scales  to mimic the house styles of The Economist and Five Thirty-Eight.  Often these can save a lot of time over trying to come up with your own color scale.